### PR TITLE
Ruby/ Resistor Duo Update mentor notes with pledge for array solution

### PR DESCRIPTION
**Pull request by F3PiX**
It's too early in the track to ask for hashes.

Drive by refactoring: fix formatting of the changelog entries

### CODE REVIEW

**Comment by F3PiX** [comment]
@exercism/website-copy-ruby

**Comment by kotp** [comment]
It is unfortunate perhaps that the instructions show the listing in a form that is extremely hinting at a key: value pair complete with the colon.  It may drive thinking that we want to work with hashes on that alone.  I think it works better as a Hash, visually, and even to work with.

File: tracks/ruby/exercises/resistor-color-duo/mentoring.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Resistor Color Duo is a side exercise, unlocked by Hello World
+Resistor Color Duo is a side exercise, unlocked by Hello World (meaning this _may_ be the first time they get mentoring!). 
 
 ### New Concepts
```
Comment (SleeplessByte) [inline tracks/ruby/exercises/resistor-color-duo/mentoring.md]:
```suggestion
### New Concepts

```

Add a new line after a heading

File: tracks/ruby/exercises/resistor-color-duo/mentoring.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 Array, Array#index, `map`, Constant, chaining methods
```
Comment (SleeplessByte) [inline tracks/ruby/exercises/resistor-color-duo/mentoring.md]:
```suggestion
`Array`, `Array#index`, `map`, Constant, chaining methods
```

File: tracks/ruby/exercises/resistor-color-duo/mentoring.md [hunk 3]
```diff
@@ -19,7 +19,9 @@ end
 ```
 
 ### General
```
Comment (SleeplessByte) [inline tracks/ruby/exercises/resistor-color-duo/mentoring.md]:
```suggestion
### General

```

Add a new line after a heading

File: tracks/ruby/exercises/resistor-color-duo/mentoring.md [hunk 4]
```diff
@@ -19,7 +19,9 @@ end
...
-This is the very first Array exercise, and the first loop. Introducing `map` is key in this exercise.
+This is the very first Array exercise, and the first loop. Introducing `map` is key in this exercise. 
+
+You, dear mentor, are totally right that using a hash would be a better solution, because it makes the relation between the color and its value explicit. But we really want people to know how to work with arrays before we ask them to work with hashes. (That will be taken care of later on in the track.) If a student _starts_ with a hash themselves, then by all means go with the hash.
 
 ### Talking points
```
Comment (SleeplessByte) [inline tracks/ruby/exercises/resistor-color-duo/mentoring.md]:
```suggestion
### Talking points

```

Add a new line after a heading

File: tracks/ruby/exercises/resistor-color-duo/mentoring.md (hunk 2 above)
Comment (F3PiX) [reply tracks/ruby/exercises/resistor-color-duo/mentoring.md]: I don't agree, but I don't want to block the merging over this.

**Comment by F3PiX** [comment]
@kotp @SleeplessByte Made the changes. If you agree, can you merge too please?

**Comment by F3PiX** [comment]
@kotp said:
> It is unfortunate perhaps that the instructions show the listing in a form that is extremely hinting at a key: value pair complete with the colon. It may drive thinking that we want to work with hashes on that alone. I think it works better as a Hash, visually, and even to work with.

This needs a new issue.

File: tracks/ruby/exercises/resistor-color-duo/mentoring.md (hunk 2 above)
Comment (SleeplessByte) [reply tracks/ruby/exercises/resistor-color-duo/mentoring.md]: :)

**Comment by SleeplessByte** [comment]
> @kotp said:
> 
> > It is unfortunate perhaps that the instructions show the listing in a form that is extremely hinting at a key: value pair complete with the colon. It may drive thinking that we want to work with hashes on that alone. I think it works better as a Hash, visually, and even to work with.
> 
> This needs a new issue.

https://github.com/exercism/problem-specifications/issues/1524

**Comment by SleeplessByte** [comment]
I believe @kotp points were addresses as well, so let's merged and continue discussion / proposal in the problem-specifications repository.

**Pull request merged**